.github/workflows/sub_lint.yml

```diff
@@ -77,7 +77,73 @@ on:
         type: boolean
         required: false
       cpplintFilters:
-        default: -whitespace-*
+        default: >-
+          '
+          +build/deprecated,
+          +build/endif_comment,
+          +build/explicit_make_pair,
+          +build/forward_decl,
+          +build/header_guard,
+          +build/include,
+          +build/include_subdir,
+          +build/include_alpha,
+          -build/include_order,
```

Comment:
Forcing alphabetical include order has already led to problems with clang-format and has been disabled: https://github.com/FreeCAD/FreeCAD/commit/b65ae56f1faa4f3d9390915178beb0bebe99a3f1

Reply:
OK -- to the best of our ability we should make the cpplint checks match the clang-tidy, so I'll disable that here as well.